src/dotty/tools/dotc/Compiler.scala

```diff
@@ -41,56 +41,58 @@ class Compiler {
    */
   def phases: List[List[Phase]] =
     List(
-      List(new FrontEnd),           // Compiler frontend: scanner, parser, namer, typer
```

Comment:
OK, will revert formatting change (though note that `git blame -w` and `git diff -w` make this less important)